### DISCUSSION TITLE
Compatibility with astro.js

### DISCUSSION THREAD

**Discussion by orenghe988** [mantinedev/mantine#1771] [Q&A]
Hey there!

I'm using Astro.js for my project, and I wanted to make sure wheather it is compatible with Mantine 'as is', and that there is no additional configuration required to use the components by Mantine other than installing the packages offered in the website.
I aim to use @mantine/core,  @mantine/hooks and @mantine/form although I might add more later down the line.

thanks in advance
Oren.

**Comment by rtivital** [comment]
We do not provide any guides for Astro.js integration, you can try it out and see if it works. Maintainers do not have any experience with it, we cannot provide any assistance.

**Comment by mapsgeek** [comment]
@orenghe988 have you found a solution to integrate it correctly?
